cmd/natdisco/main.go

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (C) 2021 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/mysteriumnetwork/node/nat/behavior"
+)
+
+var (
+	addrStrPtr      = flag.String("server", "stun.stunprotocol.org:3478", "STUN server address")
```

Comment:
Is this `StrPtr` stand for a pointer? No need to include the type of variable in the name.
```suggestion
	address      = flag.String("server", "stun.stunprotocol.org:3478", "STUN server address")
```